sdk/core/Azure.Core/README.md

```diff
@@ -1,45 +1,100 @@
 # Overview 
 
-Azure.Core is a reusable library used to implement .NET Azure SDK components. 
-It contains APIs that make it easy to call Azure service endpoints in a convinient, performant, and consistent manner.
+Azure.Core provides shared primitives, abstractions, and helpers for .NET Azure SDK service client libraries. 
+These shared APIs allow client libraries to expose common functionality in a consistent fashion, 
+so that once you learn how to use these APIs in one client library, you will know how to use them in other client libraries.
 
-# Installing
+The main shared concepts of Azure.Core (and so all Azure SDK libraries using Azure.Core) include:
```

Comment:
I would actually keep a list of libraries that use Azure.Core - otherwise it would be difficult for users to know

Reply:
I would hesitate to recommend that, given that it is very likely to get stale as more services are onboarded, make use of Core, and don't think there is a need to update it's README.

Reply:
We could add a sample list, in case people want to look at how other services did it.
At least we should call out App Configuration.

Reply:
@jsquire that's fair, but there should be at least one centralized list somewhere that we should be pointing to - without that there is no way of knowing for people which of the services implement it

Reply:
I don't disagree, though I'm not sure why knowing the consumers would be essential either.   For a given client library, Core will appear in it's list of dependencies.   When changing Core directly, breaking changes will be exposed through CI during the pull request.  Mariana's idea of highlighting a couple of "champion" SDKs would cover the case of "I want to look at a sample."

Do you have a specific scenario in mind for where the list would be useful that I'm not thinking of?  That said, I'm not opposed.  My point is only that, for it to be effective, any list would either need to be automated or maintaining it would have to be a documented and reviewed part of the process somewhere;  IMHO, a list left to rot would be more detrimental than not having one at all.

Reply:
It probably wouldn't hurt to call out that packages and repo folders prefixed with `Azure.` are following the guidelines and likely referencing `Azure.Core`.

Reply:
That sounds good to me

Reply:
I already commented on this in the past (in some other docs), but I think we need to have a public name for Track 2. We will need to refer to it in many places, not just here.

Reply:
For now, changed the first sentence to: "Azure.Core provides shared primitives, abstractions, and helpers for modern .NET Azure SDK client libraries (libraries with package names and namespaces starting with 'Azure'). "